NewController.php

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Hello World in GitHub</title>
+
+<meta name="viewport" content="width=device-width; initial-scale=1.0" />
+<meta name="author" content="Team Bah" />
+<meta name="description" content="The description of website" />
+
+<link href="bootstrap/css/bootstrap.css" rel="stylesheet" />
+<link href="bootstrap/css/bootstrap-theme.css" rel="stylesheet" />
+<link href="bootstrap/css/custom.css" rel="stylesheet" />
+
+<script src="bootstrap/js/jquery.min.js"></script>
+<script src="bootstrap/js/bootstrap.min.js"></script>
+
+</head>
+<body>
+<div class="container-fluid">
+	<div class="row" id="header">
+		<div class="col-md-12">
+			<h1>New Controller</h1>
```

Comment:
This is not a controller. This may be a view. Let us reformat our codes into codeigniter framework!

Reply:
Thank you